velero-plugins/route/restore.go

```diff
@@ -31,26 +31,32 @@ func (p *RestorePlugin) Execute(input *velero.RestoreItemActionExecuteInput) (*v
 	route := routev1API.Route{}
 	itemMarshal, _ := json.Marshal(input.Item)
 	json.Unmarshal(itemMarshal, &route)
-
-	client, err := clients.CoreClient()
-	if err != nil {
-		return nil, err
-	}
-	config, err := client.ConfigMaps("openshift-apiserver").Get("config", metav1.GetOptions{})
-	if err != nil {
-		return nil, err
-	}
-	serverConfig := common.APIServerConfig{}
-	err = json.Unmarshal([]byte(config.Data["config.yaml"]), &serverConfig)
+	version, err := common.GetServerVersion()
 	if err != nil {
 		return nil, err
 	}
+	if version.Major == "4" {
```

Comment:
I think what you want here is Major == 1 and Minor > 11

Reply:
Thanks fixed